src/pages/production/planning/page.tsx

```
@@ -0,0 +1,342 @@
+    plannedProduction: 0,
+    plannedDate: '2024-01-30',
+    priority: '높음',
+    status: '구매요청',
+    leadTime: 3,
+    bomRequired: false,
+    materialAvailability: '구매필요',
+  },
+];
+
+export default function ProductionPlanning() {
+  const navigate = useNavigate();
```

ESLint

[error] 'navigate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)